deebot_client/events/fan_speed.py

```diff
@@ -14,8 +14,10 @@ class FanSpeedLevel(IntEnum):
 
     # Values should be sort from low to high on their meanings
     QUIET = 1000
+    STANDARD = -1
```

Comment:
Can we do the same as what we did with the Lifespan enum

Reply:
@edenhaus LifeSpan was a StrEnum. Do you suggest to switch to StrEnum? 

Or do you suggest to add a method `from_xml` that is responsible for mapping?

Reply:
Implemented in 9a239b2 . But doesn't feel right, because it is now mixed int/str enum.

Reply:
> But doesn't feel right, because it is now mixed int/str enum.

It's fine as you can think of it that it's a value enum, which returns the json and XML value. For historic reason the enum value will be used for the json command and the additional property will be used for xml